.github/workflows/ci.yml

```diff
@@ -0,0 +1,167 @@
+name: CI
+
+concurrency:
+  group: core-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+on:
+  merge_group:
+  pull_request:
+  workflow_call:
```

Comment:
I think we can drop this

Reply:
I think it’s useful to have that in just in case you want to kick CI on main branch manually.

Reply:
I think you'll want `workflow_dispatch` for that - `workflow_call` is to allow this to be called from other pipelines.

Reply:
🤦